tests/bears/requirements/JuliaRequirementTest.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+import shutil
+from coalib.bears.requirements.JuliaRequirement import JuliaRequirement
+
+
+@unittest.skipIf(shutil.which('julia') is None, "Julia is not installed.")
+class JuliaRequirementTestCase(unittest.TestCase):
+
+    def test_installed_requirement(self):
+        self.assertTrue(JuliaRequirement("Lint", '', '-e').is_installed())
+
+    def test_not_installed_requirement(self):
+        self.assertFalse(JuliaRequirement('some_bad_package').is_installed())
```

Comment:
here should be
``JuliaRequirement("some_bad_package", '', '-e')``
to actually prove it works.